samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -13,11 +13,58 @@ LOG_MODULE_REGISTER(raw_tx_packet, CONFIG_LOG_DEFAULT_LEVEL);
 
 #include <nrfx_clock.h>
 #include <zephyr/kernel.h>
+#include <zephyr/net/socket.h>
+
+#include <fmac_structs.h>
 
 #include "net_private.h"
 
 int try_wifi_connect(void);
 
+struct beacon {
+	uint16_t frame_control;
+	uint16_t duration;
+	uint8_t da[6];
+	uint8_t sa[6];
+	uint8_t bssid[6];
+	uint16_t seq_ctrl;
+	uint8_t payload[256];
+} __packed;
+
+static struct beacon test_beacon_frame = {
+	.frame_control = htons(0X8000),
+	.duration = 0X0000,
```

Comment:
Will this be filled by MAC? AFAIK, it has to be filled by application based on the rate. Without this we might get interference from others as their NAV isn't updated.

Reply:
In this case, I do not think it is necessary to update duration field as we are sending out beacon frames.

Reply:
For beacon its fine, but the sample should demonstrate how to do this for others as well in case customers want to send data frames. Anyways, we can take this up later.